### DISCUSSION TITLE
Timepicker ok button not showing when select a time. but when i hover on it the button showing again. how to fix it

### DISCUSSION THREAD

**Discussion by opuopu** [ant-design/ant-design#48887] [General]
when i select a time the ok button going to invisible. you can see it here 
![image](https://github.com/ant-design/ant-design/assets/43672649/63363798-4fc3-48e9-a5eb-63b272b0373b)

and when hover on it the background color showing
![image](https://github.com/ant-design/ant-design/assets/43672649/8690a296-1809-4e57-bede-3c9f9a8320c2)
im using ant design with react js. i used configprovider for theme editor.

**Comment by afc163** [comment]
Could you provide a online reproduce?

**Reply by opuopu** [reply]
check this video here.
[screen-capture (8).webm](https://github.com/ant-design/ant-design/assets/43672649/880b7fbe-70a3-456c-b6c8-8fd52df137db)
generally its not happening. but for customization i use configprovider. is there any chance it can be happend form here?

**Reply by afc163** [reply]
Could you provide a reproduce codesandbox?

**Reply by opuopu** [reply]
here: you can see a re usable timepicker in form. https://codesandbox.io/p/devbox/s54yrj?file=%2Fsrc%2Fpages%2FVendorDashboard%2FSubAdmin%2FCreateSubAdmin.tsx%3A70%2C28


click on the create sub admin button then a modal will be poped up. then you can see the timepicker in the form.

**Reply by opuopu** [reply]
and now i see its working on the sanbox devbox .  what happend. but its not working on my pc and browsers.
the same code i hosted on vercel its not working. you can see it here https://danny-booking-dashboard.vercel.app/admin/subadmin

**Reply by afc163** [reply]
<img width="1782" alt="图片" src="https://github.com/ant-design/ant-design/assets/507615/ebfcc517-2b76-476f-8910-4a980d446f81">

**Reply by afc163** [reply]
We need a codesandbox reproduce with code to see what is happenning.
